src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -115,10 +115,16 @@
           </GroupBox>
 
             <GroupBox x:Uid="ClassificationsGroupBox"
-                      Header="{x:Static local:AdvancedOptionPageStrings.Option_Classifications}">
+                      Header="{x:Static local:AdvancedOptionPageStrings.Option_Editor_Color_Scheme}">
                 <StackPanel>
-                    <CheckBox x:Name="Use_enhanced_colors"
-                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Use_enhanced_colors_for_C_and_Basic}" />
+                    <ComboBox x:Name="Editor_color_scheme" 
+                              IsEditable="false">
+                        <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Color_Scheme_Enhanced}" Tag="{x:Static local:AdvancedOptionPageStrings.Color_Scheme_Enhanced_Tag}" />
+                        <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Color_Scheme_VisualStudio2017}" Tag="{x:Static local:AdvancedOptionPageStrings.Color_Scheme_VisualStudio2017_Tag}" />
+                    </ComboBox>
+                    <TextBlock x:Name="Custom_VS_Theme_Warning"
```

Comment:
Perhaps we could just hide the combobox instead of disabling it and show the warning.
![image](https://user-images.githubusercontent.com/611219/70838666-36e21780-1dbe-11ea-851a-0ca5d68396e9.png)